assets/sass/scaffolding/_content.scss

```diff
@@ -90,11 +86,6 @@
         @include mq('medium-major') {
             flex: 0 0 320px;
         }
-
-        // Prevent sidebar from stretching to match primary content height
-        &.content-sidebar__secondary--minimal {
```

Comment:
Does removing this re-introduce the lengthy sidebar I previewed [here](https://github.com/biglotteryfund/blf-alpha/pull/1547#discussion_r241368176)?

Reply:
Ah no it doesn't because the card component now does the background colour, not the sidebar. Carry on!